LIBITD-2530. Routes for IP Manager API

aatishgupta25 opened the pull request:
Added api-description.md to ipmanager-django/docs. Included details about the 3 different API routes.

https://umd-dit.atlassian.net/browse/LIBITD-2530

Commit:
LIBITD-2530. Routes for IP Manager API

Added api-description.md to ipmanager-django/docs. Included details
about the 3 different API routes.

https://umd-dit.atlassian.net/browse/LIBITD-2530

peichman-umd reviewed (changes requested):
Looks good. One style extra to add.

For bits of code or things like variable names or data structure keys, you can use backticks (`) to highlight these as literal values. For example:

```markdown
Takes two parameters `ip` (String, Required; IP address
to be checked for membership) and `group` (String; Key
of the group)
```

Results in:

Takes two parameters `ip` (String, Required; IP address to be checked for membership) and `group` (String; Key of the group)

Commit:
LIBITD-2530. Updated Description for IP Manager API

Updated api-description.md based on review.

https://umd-dit.atlassian.net/browse/LIBITD-2530
